### ISSUE TITLE
Support `window/showMessageRequest` in the playground

### ISSUE THREAD

**Issue by decorator-factory** [DetachHead/basedpyright#1640]
### Description

https://github.com/DetachHead/basedpyright/pull/1514 fixed pyright's bug where the "rename" feature accepts very dubious symbol names like `yield` or `42 bananas`. If that is detected the language server produces a `window/showMessageRequest` thingy. The playground doesn't seem to support them and shows this in the console:

<img width="558" height="48" alt="Image" src="https://github.com/user-attachments/assets/eaea483c-9819-49d1-b5e8-29dff870e92b" />

Not sure what a good way to handle this would be. Maybe a [popover](https://web.dev/learn/css/popover-and-dialog#manual_popovers), or even an `alert()` (as 1999 as that sounds, that's a pretty rarely encountered feature in the playground)

---

Steps to reproduce:

1. Go to https://basedpyright.com
2. Enter this code

    ```py
    def foo(): pass
    ```

3. put the cursor on `foo` and press F2
4. enter "yield" and press Enter
5. no popup, see error message in the JavaScript console
